Main.java

```diff
@@ -9,7 +11,24 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Create Scanner for user input 
+     Scanner input = new Scanner(System.in);  
+
+     // Prompt the user to write down a integer 
+     System.out.println("Please enter an integer:");
+     int Integer_1 = input.nextInt(); 
+
+     // prompt the user to enter another integer 
+     System.out.println("Please enter another integer:"); 
+     int Integer_2 = input.nextInt(); 
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. So you can call them integer1 and integer2. That would work just fine.